### PR TITLE
Minutes from 2019-07-11 meeting

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
Had to reformat the Reco & Software Triggers quite a lot as the formatting did not translate well from Google Doc

### CODE REVIEW

File: organization/_posts/2019/2019-07-11-coordination.md [hunk 1]
```diff
@@ -0,0 +1,313 @@
+---
+title: "HSF Weekly Meeting #168, 20 June, 2019"
```
Comment (eduardo-rodrigues) [inline organization/_posts/2019/2019-07-11-coordination.md]: This line needs to be updated.

File: organization/_posts/2019/2019-07-11-coordination.md [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+Pere Mato, Agnieszka Dziurda, Daniel Elvira, Andrei Gheata, Serhan Mete,
+David Lange, Ed Moyse, Paul Laycock, Torre Wenaus, Ben Morgan, Dario
+Menasce, Eduardo Rodrigues, Gloria Corti, Liz-Sexton Kennedy, Heather
+Gray, Sudhir Malik, Caterina Doglioni, Mark Neubauer
+
+*Apologies*: Andrea Valassi
+
+## News, general matters
+  - HPCs and GPUs - cost model and benchmarking WG
+      - WLCG MB has set up a group to work on benchmarking of GPUs in
```
Comment (eduardo-rodrigues) [inline organization/_posts/2019/2019-07-11-coordination.md]: Many acronyms here. I would spell out MB and FA.

File: organization/_posts/2019/2019-07-11-coordination.md [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+        with HPCs, dynamic cloud resources, other non-standard
+        resources)
+      - But it’s actually backend agnostic, so a request to host the
+        code as an HSF project.
+          - Part of Intelligent Data Delivery System (iDDS).
+      - N.B. similar to things we currently do host, like CREST and
+        Phoenix.
+      - Agreed.
+  - Summer break now, when to resume meetings?
+      - Resume 5 September.
```
Comment (eduardo-rodrigues) [inline organization/_posts/2019/2019-07-11-coordination.md]: Try and update the date if we converge. In any case it would either be Aug 29 or Sep 12.

**Comment by graeme-a-stewart** [comment]
CI failure is the usual (and known) coda lab outage, otherwise all good

**Comment by graeme-a-stewart** [comment]
Thanks @eduardo-rodrigues - I'll wait a little while to see if others review, otherwise I'll merge.

**Comment by graeme-a-stewart** [comment]
Thanks both! Merged

**Pull request merged**